### PR TITLE
Changed 'Simple Buttons' to 'Buttons' throughout

### PR DESCRIPTION
**Pull request by chri2547**
Changed a ton of file names.

### CODE REVIEW

File: _component_design/selector-strip.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Consider the following best practices when you construct a selector strip:
 - Use short option labels (for example, three words maximum and up to 10 characters) so that the number of options can fit horizontally on a small screen.
 - If they are commonly understood, use abbreviations. For example, on a credit card selector strip the user understands that *AmEx* stands for *American Express*.
 - A selector strip should not wrap.
-- Do not use a selector strip to trigger an action. If you need multiple triggers alongside each other, consider a [button bar]({{site.baseurl}}/components/simple-buttons.html) or a [button set]({{site.baseurl}}/components/button-bar.html).
+- Do not use a selector strip to trigger an action. If you need multiple triggers alongside each other, consider a [button bar]({{site.baseurl}}/components/buttons-bar.html) or a [button set]({{site.baseurl}}/components/buttons.html).
```
Comment (nimbinatus) [inline _component_design/selector-strip.md]: Just to check, the switch here was intentional? From `button bar` pointing to `simple-buttons.html` and `button set` pointing to `button-bar.html` to `button bar` pointing to `button-bar.html` and `button set` pointing to `buttons.html`?

File: _component_design/buttons.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ usage: >
 preview-image: preview-images/buttons.svg
 status: stable
 resource: true
-last-modified: 2018-08-27
+last-modified: 2018-09-27
 helix-ui-css: true
 helix-ui-javascript: na
 pagelink: https://rackerlabs.github.io/helix-ui/components/buttons/#basic-button
```
Comment (nimbinatus) [inline _component_design/buttons.md]: On line 29, please change `site.baseURL` to site.baseurl` while we're in here.

File: _component_design/buttons-bar.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ Use a button bar in the following situations:
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/buttons-bar.md]: Done.

File: _component_design/buttons.md (hunk 2 above)
Comment (chri2547) [reply _component_design/buttons.md]: Done.

File: _component_design/selector-strip.md (hunk 1 above)
Comment (chri2547) [reply _component_design/selector-strip.md]: Correct, that change was intentional.

**Pull request merged**